cli/build.go

```diff
@@ -704,17 +697,14 @@ func (b BuildData) EnvironmentVariables() []string {
 
 // TODO: non-linux things too
 func DownloadYB() (string, error) {
-	downloadUrl := "https://bin.equinox.io/a/7G9uDXWDjh8/yb-0.0.39-linux-amd64.tar.gz"
+	// Stable version stable URL
+	downloadUrl := "https://bin.equinox.io/c/3fw6ZrH8UFD/yb-stable-linux-amd64.tgz"
 	currentPath, _ := filepath.Abs(".")
 	tmpPath := filepath.Join(currentPath, ".yb-tmp")
-	MkdirAsNeeded(tmpPath)
 	ybPath := filepath.Join(tmpPath, "yb")
-	if _, err := os.Stat(ybPath); err == nil {
-		return ybPath, nil
-	}
+	MkdirAsNeeded(tmpPath)
```

Comment:
Did you mean to change this to ybPath, since you moved the call to after setting that variable?

Reply:
Actually tmpPath == path to the temporary directory that will keep the yb binary; ybPath == tmpPath + "/yb", right?

Reply:
Ridai is correct -- tmpPath is where we un-archive the binary to, so we need to make it before we explode the archive